README.md

```diff
@@ -1,70 +1,88 @@
-
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## List of Moderators
 
-### `yarn test`
+* **[Mayank Talwar](https://github.com/DreadArceus)**
+* **[Akash Mondal](https://github.com/AkMo3)**: GitHub
+* **[Om Gupta](https://github.com/OmGuptaIND)**: React, Node.js, Django REST, MongoDB, PostgreSQL, SQLite
+* **[Souhardya Malakar](https://github.com/SouhardyaMalakar)**: CSS, React
+* **[Arush Kurundodi](https://github.com/arcinston)**
+* **[Sriraj Behera](https://github.com/SrirajBehera)**: JavaScript, Node, MongoDB
+* **[Utkarsh Choubey](https://github.com/utk2541)**
+* **[Ananjan](https://github.com/ananjan)**
+* **[Shivral](https://github.com/shivral)**: Web Scraping
+* **[Siddhant Singhania](https://github.com/sid11428)**
+* **[Sourabh](https://github.com/sourabhjo7)**
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
 
-### `yarn build`
+## Members of the LNMIIT-CODING STONKS
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+* [Akash Mondal](https://github.com/AkMo3)
+* [Ananjan](https://github.com/ananjan)
+* [Ansh Pant](https://github.com/AnshPant)
+* [Arush Kurundodi](https://github.com/arcinston)
+* [Astha Singh](https://github.com/Astha-Singh1729)
+* [Ayush Shukla](https://github.com/Ayush-Shukla02)
+* [Mayank Talwar](https://github.com/DreadArceus)
+* [MaySi11](https://github.com/MaySi11)
+* [Pranav Mehta](https://github.com/mehta1351)
+* [N-Coder-Bot](https://github.com/N-coder-bot)
+* [Om Gupta](https://github.com/OmGuptaIND)
+* [Shivral](https://github.com/shivral)
+* [Siddhant Singhania](https://github.com/sid11428)
+* [Souhardya Malakar](https://github.com/SouhardyaMalakar)
+* [Sourabh](https://github.com/sourabhjo7)
+* [Sriraj Behera](https://github.com/SrirajBehera)
+* [Utkarsh Choubey](https://github.com/utk2541)
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+# Setting up Project
 
-### `yarn eject`
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+# Contribution
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+## Getting Started with Create React App
```

Comment:
Adds the steps of contribution here shared with you [here](https://github.com/LNMIIT-CODING/stonks-frontend/issues/1#issuecomment-804931213)

Reply:
Done